src/components/Candidates.tsx

```diff
@@ -51,14 +58,16 @@ const WithCursorBlock = withCursor(ImageTextBlock);
 
 const Candidates: React.FC = () => {
   const listOfBlocks = candidates.map(candidate => 
-    <WithCursorBlock name={candidate} key={candidate}/>
+    <Box key={candidate}>
+      <WithCursorBlock name={candidate}/>
+    </Box>
   );
   return (
     <>
       <Div>Candidates
-        <div className="wrapper">
+        <Flex className="wrapper" flexDirection={['column', 'row']} justifyContent={['space-around']}>
```

Comment:
justifyContent={['space-around']} - it should be simply justifyContent="space-around"
If you need only string as a prop use just string